src/model/element-base.ts

```diff
@@ -107,4 +120,4 @@ export abstract class ElementBase implements Feature {
   emitEventMetadata(_event: Event): Object {
     return {};
   }
-}
+}
```

Comment:
newline at end of file. recommend configuring your editor to automatically add this